src/Sentry.Unity.Android/AndroidJavaScopeObserver.cs

```diff
@@ -34,76 +26,49 @@ public void AddBreadcrumb(Breadcrumb breadcrumb)
             sentry.CallStatic("addBreadcrumb", javaBreadcrumb, null);
         }
 
-        public void SetExtra(string key, object? value)
+        public override void SetExtraImpl(string key, string? value)
         {
             AndroidJNI.AttachCurrentThread();
-
-            _options.DiagnosticLogger?.LogDebug("Android Scope Sync - Setting Extra k:\"{0}\" v:\"{1}\"", key, value);
-
-            string? extraValue = null;
-            if (value is not null)
-            {
-                extraValue = SafeSerializer.SerializeSafely(value);
-                if (extraValue is null)
-                {
-                    return;
-                }
-            }
-
-            using var sentry = GetSentryJava();
-            sentry.CallStatic("setExtra", key, extraValue);
+            GetSentryJava().CallStatic("setExtra", key, value);
```

Comment:
Sematics changed here because the Java objects need to be disposed. Before we had:

**using** var sentry = GetSentryJava();

Now the dispose is not called.

We need to go back to:

```
            using var sentry = GetSentryJava();
            sentry.CallStatic("setExtra", key, extraValue);
```

Reply:
I see... somehow my mind (non-c# developer) hasn't expected `using` to do anything like that... I'll roll back the change, thanks